doc/matchers.md

```diff
@@ -90,6 +90,10 @@ For the extension function style, each function has an equivalent negated versio
 | `double.shouldBeLessThanOrEqual(n)` | Asserts that the double is less or equal to than the given value n |
 | `double.shouldBeGreaterThan(n)` | Asserts that the double is greater than the given value n |
 | `double.shouldBeGreaterThanOrEqual(n)` | Asserts that the double is greater than or equal to the given value n |
+| `double.shouldBePositive()` | Asserts that the double is positive |
+| `double.shouldBeNotPositive()` | Asserts that the double is not positive|
+| `double.shouldBeNegative()` | Asserts that the double is negative |
+| `double.shouldBeNotNegative()` | Asserts that the double is not negative |
```

Comment:
Same as the positive case, I believe you're going for `double.shouldNotBeNegative`

Reply:
oh that's correct, my bad